app/components/AppUpdater/AppUpdater.js

```diff
@@ -1,16 +1,28 @@
 // @flow
 import React from 'react'
-import { ipcRenderer } from 'electron'
 
-import appUpdateModal from '../../services/appUpdateModal'
+import { checkForUpdates } from '../../services/app-update'
+
+import appUpdateModal from './AppUpdateModal'
+
+const POLL_INTERVAL = 1000 * 5 * 60
+const pollForUpdates = async () => {
+  const updateLink = await checkForUpdates()
+  if (updateLink) {
+    const cancel = await appUpdateModal(updateLink)
+    if (cancel) {
+      setTimeout(pollForUpdates, POLL_INTERVAL)
```

Comment:
Won't this nag the user every POLL_INTERVAL even if he declines the suggested update?

Reply:
@thehogfather @goldylucks 

Lets add a checkbox that says "Don't ask me again"

If the user doesn't click the checkbox make the interval every 30 Minutes.